docs/source/basic.rst

```diff
@@ -116,6 +116,63 @@ notify their running by creating a .pid file.
 
 
 
+Timeouts
```

Comment:
Maybe we should enhance timeout definition for docblock: http://mirakuru.readthedocs.org/en/v0.5.0/api/executors.html#mirakuru.base.Executor

Also, I don;t think there's a need to list how to use timeout in all executors, since it's basically the same usage, and same behaviour. Plus maybe we should separate Timeout header and Context manager to separate doc (advanced, or simply usage?), and keep basic as simple list of executors.